lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java

```diff
@@ -33,55 +33,46 @@ public static void GlobalTearDown() {
     }
 
     @Test
-    public void sum() {
-        int a = 4;
-        int b = 3;
-
-        int expected = 7;
-        int actual = calculator.sum(a, b);
-
+    public void testDivision() {
+        int a = 12;
+        int b = 4;
```

Comment:
try with b= 0,

Reply:
I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files.